design/kata-api-design.md

```diff
@@ -23,17 +23,24 @@ To fulfill the [Kata design requirements](kata-design-requirements.md), and base
 
 |Name|Description|
 |---|---|
-|`sandbox.Pause()`| Pause the sandbox.|
-|`sandbox.Resume()`| Resume the paused sandbox.|
-|`sandbox.Release()`| Release a sandbox data structure, close connections to the agent, and quit any goroutines associated with the sandbox. Mostly used for daemon restart.|
 |`sandbox.Delete()`| Destroy the sandbox and remove all persistent metadata.|
-|`sandbox.Status()`| Get the status of the sandbox and containers.|
 |`sandbox.Monitor()`| Return a context handler for caller to monitor sandbox callbacks such as error termination.|
+|`sandbox.Release()`| Release a sandbox data structure, close connections to the agent, and quit any goroutines associated with the sandbox. Mostly used for daemon restart.|
+|`sandbox.Start()`| Start a a sandbox and the containers making the sandbox.|
+|`sandbox.Status()`| Get the status of the sandbox and containers.|
+|`sandbox.Stop()`| Stop a sandbox and Destroy the containers in the sandbox..|
```

Comment:
Nit: lowercase "destroy" (otherwise suggests it's a method name)

Reply:
In doing reviews like this, you realize how stupid the Go convention of capitalizing public methods really is. It's really pointlessly painful to find the exports in a file from an editor. Even `guru` does not help. Any good suggestion there?

Reply:
Added #729 about missing documentation for types.

Reply:
Hi @c3d, I don't have any means about the problem of Go. Thanks Christophe, the types are important for the api docs. I will fix the #729 issue.

Reply:
@c3d - I tend to either use a crazy grep regex, but also have a golang program that uses ast + parse packages to list functions (it "works" but doesn't list the receiver for methods yet :cry:)